src/SARIF.hs

```diff
@@ -0,0 +1,177 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+
+{- |
+Description: Formats hlint ideas in the Statis Analysis Results Interchange Format (SARIF).
+License: BSD-3-Clause
```

Comment:
Any reason to add a license to just this one file, given the entire project is BSD3 licensed?

Reply:
I fill out Haddock module documentation fields out of habit, and the license field was just something that could be filled.  Other than the fact that I wrote these lines, there's no reason this file needs special treatment.